tests/integration/s2n_tls13_handshake_tests.py

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+"""
+Handshake tests against openssl using TLS13.
+At the moment these tests are expected fail, as TLS13 is incomplete.
+"""
+
+import argparse
+import os
+import sys
+from multiprocessing.pool import ThreadPool
+from s2n_openssl_common import *
```

Comment:
run_scenarios and get_scenarios are both in s2n_test_common, perhaps s2n_test_common should be imported here, and s2n_openssl_common should be imported from test_common?